example/android/app/src/main/AndroidManifest.xml

```diff
@@ -15,18 +15,20 @@
       <meta-data
         android:name="com.google.android.gms.wallet.api.enabled"
         android:value="true" />
+      <activity android:name=".LaunchActivity" android:exported="true">
```

Comment:
To handle deeplink `.LaunchActivity` should be `singleTop` otherwise it may cause two instances of the app

```suggestion
      <activity android:launchMode="singleTop" android:name=".LaunchActivity" android:exported="true">
```

Reply:
The trick with this split into two activities is that deeplinks should be handled by MainActivity directly, not LaunchActivity. MainActivity can then be singleTask or singleTop depending on what you prefer.

Reply:
Yes, and kept deeplinks handled by `MainActivity` with `singleTask` launchMode (required to prevent issue with 3DS) will cause multiple instances. Using `android:launchMode="singleTop"` for `LaunchActivity` will prevent the multiple instances, as the deeplink will route the intent to the existing instance instead of creating a new one.

But I agree that it may depend on the use case you have to handle deeplink. This change is not required, but may be necessary depending on the project configuration.